pages/02.blog/03.announcements/tutorials-in-English/post.fr.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Tutoriels en anglais'
+
+date:   22/6/2022
+
+taxonomy:
+    category: [blog, announcements]
+    tag: [tutorials]
+    author: 'Eric Laporte'    
+
+markdown:
+    extra: true
+    auto_url_links: true
+
+---
+
+Des tutoriels en anglais sur Unitex/GramLab sont disponibles.
+
+#### Prise en main
+
+- [Diaporama](https://unitexgramlab.org/blog/announcements/tutorials-in-english/local-grammar-graphs.pdf)
+- [Exercices](https://unitexgramlab.org/blog/announcements/tutorials-in-english/getting-started.pdf)
+
+#### Reconnaissance d'entités nommées
+
+- [Préparation](https://unitexgramlab.org/blog/announcements/tutorials-in-english/Unitex_install.pdf)
+- [Diaporama](https://unitexgramlab.org/blog/announcements/tutorials-in-english/Unitex_NER.pdf)
+- [Exercices](https://unitexgramlab.org/blog/announcements/tutorials-in-english/Unitex_NER_exercises.pdf)
```

Comment:
Even if possible, the code repository is not designed to version/host binary documents, including .PDFs, there are sites such as https://www.docdroid.net (you can take another one) that allow to share PDFs efficiently , see e.g: https://docdro.id/3QHeKxj

In this way: 

- [Exercices](https://unitexgramlab.org/blog/announcements/tutorials-in-english/Unitex_NER_exercises.pdf)
```
- [Exercices](https://unitexgramlab.org/blog/announcements/tutorials-in-english/Unitex_NER_exercises.pdf)
```

May be:

- [Exercices](https://docdro.id/3QHeKxj)
```
- [Exercices](https://docdro.id/3QHeKxj)
```